.circleci/config.yml

```diff
@@ -0,0 +1,62 @@
+version: 2.1
+
+workflows:
+  ruby-tests:
+    jobs:
+      - unit-tests:
+          matrix:
+            parameters:
+              version: ["2", "2.1", "2.2", "2.3", "2.4", "2.5", "2.6.5", "2.7"]
+      - unit-tests-legacy:
+          matrix:
+            parameters:
+              version: ["kneip/ree-1.8.7-2012.02","ruby:1.9.3","circleci/jruby:9"]
+
+orbs:
+  ruby: circleci/ruby@0.1.2
+
+jobs:
+  unit-tests:
+    parallelism: 1
+    parameters:
+      version:
+        type: string
+    docker:
+      - image: circleci/ruby:<< parameters.version >>
+    steps:
+      - checkout
+      - run:
+          name: Versions
+          command: |
+            echo "ruby: $(ruby --version)"
+      - run:
+          name: Install dependencies
+          command: bundle install
+      - run:
+          name: Run tests
+          command: bundle exec rake spec
+
+  unit-tests-legacy:
+    parallelism: 1
+    environment:
+      BUNDLE_GEMFILE: ./gemfiles/Gemfile.legacy
+    parameters:
+      version:
+        type: string
+    docker:
+      - image: << parameters.version >>
+    steps:
+      - checkout
+      - run:
+          name: Versions
+          command: |
+            echo "ruby: $(ruby --version)"
+      - run:
+          name: Install dependencies
+          command: |
+            gem install bundler --version 1.8.7
```

Comment:
We want this version to be anything < 2.0, in Travis it looks like it was `1.16.6` but I think `1.17.3` would be great.

Reply:
gotcha, will try it